test/py/SerializationTest.py

```diff
@@ -285,6 +290,67 @@ def testIntegerLimits(self):
         for value in bad_values:
             self.assertRaises(Exception, self._serialize, value)
 
+    def testRecTree(self):
+        """Ensure recursive tree node can be created."""
+        children = []
+        for idx in range(1, 5):
```

Comment:
This is just a test to ensure you can make a tree with some children.  There is no limit on the number of recursions (I guess besides the maximum allowed by python).